native/cocos/bindings/jswrapper/napi/Object.cpp

```diff
@@ -502,6 +502,8 @@ void Object::weakCallback(napi_env env, void* nativeObject, void* finalizeHint /
         if (nativeObject == nullptr) {
             return;
         }
+        const char* clsName = obj->_getClass()->getName();
+        LOGI("weakCallback class name:%{public}s, ptr:%{public}p", clsName, nativeObject);
```

Comment:
Debug codes?

Reply:
yes

Reply:
It is better add TODO, then should remove debug codes before releasing.